core/storage/suggestion/gae_models.py

```diff
@@ -726,6 +726,27 @@ def get_in_review_translation_suggestions_with_exp_ids_by_offset(
             next_offset
         )
 
+    @classmethod
+    def get_in_review_translation_suggestions_by_exp_id(
+        cls, exp_id: str
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all in-review translation suggestions matching the supplied
+        exp_id.
+
+        Args:
+            exp_id: str. Exploration ID matching the target ID of the
+                translation suggestions.
+
+        Returns:
+            list(SuggestionModel). A list of suggestions matching the supplied
+            exp_id.
+        """
+        return cls.get_all().filter(datastore_services.all_of(
+            cls.status == STATUS_IN_REVIEW,
+            cls.suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            cls.target_id == exp_id
+        )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
```

Comment:
Hmm, I did test this locally, but there were no `index.yaml` changes. Historically, I've had this issue, so it may be something to do with my personal setup.

Reply:
Hmm, you might need to update index.yaml manually then... 

@vojtechjelinek is there a way for e.g. someone else to pull the code, run some script, and generate the necessary changes that should be made? We might want to document that (and/or maybe have some sort of standard diagnostic test that devs can run to see whether this works on their machine -- it could work by asking them to add a function with a query, running the script, and seeing if index.yaml changes).

Reply:
Updated the index manually for now.

Reply:
@seanlip It is pretty much what you said, pull, run and see if the index.yaml is updated. Not sure where or how to document this.

Reply:
@vojtechjelinek I would put it in a wiki page titled "Things to keep in mind when making storage layer changes" or similar. Filed https://github.com/oppia/oppia-web-developer-docs/issues/122 to track.